### PR TITLE
sort results in ruby for efficiency

### PR DESCRIPTION
**Pull request by cantino**
@bennapp we forgot to submit this.

### CODE REVIEW

**Comment by cantino** [comment]
@jhmoore @shirish-pampoorickal Review please

File: lib/brainstem/presenter_collection.rb [hunk 1]
```diff
@@ -79,7 +79,10 @@ def presenting(name, options = {}, &block)
 
         # On complex queries, MySQL can sometimes handle 'SELECT id FROM ... ORDER BY ...' much faster than
         # 'SELECT * FROM ...', so we pluck the ids, then find those specific ids in a separate query.
-        new_scope = handle_ordering(scope.klass.where(id: scope.pluck("#{scope.table_name}.id")), options)
+        ids = scope.pluck("#{scope.table_name}.id")
+        id_lookup = {}
+        ids.each.with_index { |id, index| id_lookup[id] = index }
+        new_scope = scope.klass.where(id: id_lookup.keys).sort_by { |model| id_lookup[model.id] }
```
Comment (shirish-pampoorickal) [inline lib/brainstem/presenter_collection.rb]: new_scope variable seems unnecessary

File: lib/brainstem/presenter_collection.rb [hunk 2]
```diff
@@ -79,7 +79,10 @@ def presenting(name, options = {}, &block)
...
         records = new_scope.to_a
```
Comment (shirish-pampoorickal) [inline lib/brainstem/presenter_collection.rb]: Also, now that we have sort_by, I don't think `to_a` is necessary

**Comment by shirish-pampoorickal** [comment]
LGTM other than the unnecessary variable & `to_a`

**Comment by cantino** [comment]
Thanks @shirish-pampoorickal!

**Pull request merged**